dangerfile.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+
+function eslintReporter() {
+  const eslintOutput = './eslint.json';
+
+  if (!fs.existsSync(eslintOutput)) {
+    return;
+  }
+
+  const output = JSON.parse(fs.readFileSync(eslintOutput).toString());
+
+  output.forEach(items => {
+    if (items.messages.length) {
+      items.messages.forEach(item => {
+        const reportLevel = item.severity === 2 ? fail : warn;
```

Comment:
<!--
  2 failure:  'fail' is not def..., 'warn' is not def...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: dangerfile.js;
  Line: 15;
-->  
- :no_entry_sign: 'fail' is not defined.
- :no_entry_sign: 'warn' is not defined.